952 loading performance graphql

lorenjohnson opened the pull request:
Staging is current to this branch and I'm still doing some QA style finger testing there. Please do to if you get to it.  Code is ready for review.

tibetsprague reviewed (approved):
so the main thing is just not loading the group data with the me query? sweet :)

lorenjohnson commented:
> so the main thing is just not loading the group data with the me query? sweet :) 


Sorta--because of the format change it’s not so apparent in the diff, but I removed a number of fields from that initial query. 

Most importantly from the ‘memberships { groups }’ I removed ‘memberCount’ which was having an outsized impact and ‘groupTopics’. Both after timing with and without through multiple runs of the query in isolation using graphical.

Regarding the out-sized impact of  ‘memberCount‘, I got a notable bump just leaving that out of the test query. It's apparent why this is the case when you take a peek at ‘Group#memberCount’ in node, I’ll get put in a ticket for that tomorrow.

Splitting the current group load to happen in parallel with me was the rest of it but surprisingly not necessarily the majority of the gain.  I’d have split the abbreviated memberships into a separate parallel loading query but the gain was negligible after removing the big offenders which luckily were both unnecessary as they are not on the initial screens and when they are they’re already being re-fetched in those contexts. Still gonna do a little more testing on it tomorrow to  double confirm that conclusion tomorrow before releasing it.

Will be curious how it performs in production. (Seems to be my week for long responses 🤣🙏🏼)

thomasgwatson reviewed (approved):
lgtm

lorenjohnson commented on graphql.config.json:
```diff
@@ -0,0 +1,11 @@
+{
```
That and the `@playlyfe/gql` dependency added are about giving the VS Code GraphQL extension what it needs to find and reference our GraphQL schema for schema inline validation and autocomplete when writing GraphQL. The extension is optional but handy.
